### PR TITLE
Basic rich text styles for list and blockquotes in notes

### PR DESCRIPTION
**Pull request by alebedev**
## What?
Add basic styles for lists and block quotes in rich text notes

Using CSS modules because we need to pass down classname, emotion won't do it for us AFAIK

## Why?
Improve on basic browser styles, especcially for editor preview

## Optional checklist
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1212?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #1208** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1208?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #1209** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1209?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #1210** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1210?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #1212** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1212?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1212?utm_source=stack-comment).

File: src/portals/hope/features/claims/claim-details/ClaimNotes/ClaimNoteItem.tsx [hunk 1]
```diff
@@ -39,9 +40,9 @@ export const ClaimNoteItem: React.FC<{
     insertedBy?.__typename === 'MemberSystemUser'
   const noteType: 'text' | 'markdown' = isSystemNote ? 'text' : 'markdown'
   let noteText = <NoteText>{text}</NoteText>
-  if (noteType == 'markdown') {
+  if (noteType === 'markdown') {
     noteText = (
-      <div>
+      <div className={css.richText}>
         <Markdown>{text}</Markdown>
       </div>
```
Comment (guilhermespopolin) [inline src/portals/hope/features/claims/claim-details/ClaimNotes/ClaimNoteItem.tsx:47]: Can we pass the `className` to `Markdown` directly?
```suggestion
    <Markdown className={css.richText}>{text}</Markdown>
```

**Review by guilhermespopolin — APPROVED** [review]
Looks good :)

File: src/portals/hope/features/claims/claim-details/ClaimNotes/ClaimNoteItem.tsx [hunk 2]
```diff
@@ -39,9 +40,9 @@ export const ClaimNoteItem: React.FC<{
     insertedBy?.__typename === 'MemberSystemUser'
   const noteType: 'text' | 'markdown' = isSystemNote ? 'text' : 'markdown'
   let noteText = <NoteText>{text}</NoteText>
-  if (noteType == 'markdown') {
+  if (noteType === 'markdown') {
     noteText = (
-      <div>
+      <div className={css.richText}>
```
Comment (falklinus) [inline src/portals/hope/features/claims/claim-details/ClaimNotes/ClaimNoteItem.tsx:45]: pretty sure it's fine to define it as nested css in emotion string css at least

**Review by falklinus — APPROVED** [review]
lgtm, just a comment about the module.css

**Comment by falklinus** [comment]
Do you think we could use a component to be able to do richtext anywhere? Or is it fine to use the MDX wherever we need it?

File: src/portals/hope/features/claims/claim-details/ClaimNotes/ClaimNoteItem.tsx (hunk 1 above)
Comment (alebedev) [reply src/portals/hope/features/claims/claim-details/ClaimNotes/ClaimNoteItem.tsx:47]: Markdown does not produce root element, so we need to insert our own

**Comment by alebedev** [comment]
### Merge activity

* **Jan 29, 10:04 AM**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1212) rebased this pull request after merging its parent, because this pull request is set to merge when ready.

**Pull request merged**